.github/CONTRIBUTING.md

```diff
@@ -124,8 +124,9 @@ In addition, linters can help to enforce coding standards, find unused variables
 
 **HospitalRun** uses the following linters:
 
-1. [ESLint](http://eslint.org/) for ECMAScript/JavaScript. You can find the ESLint User guide [here](http://eslint.org/docs/user-guide/).
+1. [ESLint](http://eslint.org/) for ECMAScript/JavaScript. You can find the ESLint User guide [here](http://eslint.org/docs/user-guide/). In addition to the default set of rules ESLint is setup to use eslint-plugin-ember-suave you can find more information about that [here](https://github.com/DockYard/eslint-plugin-ember-suave).
 2. [stylelint](http://stylelint.io/) for Stylesheets. You can find stylelint User guide [here](http://stylelint.io/user-guide/).
```

Comment:
Can you correct the grammar here to You can find _the_ stylelint user guide here